src/dmd/cli.d

```diff
@@ -88,19 +106,60 @@ struct Usage
             "generate code for all template instantiations"
         ),
         Option("betterC",
-            "omit generating some runtime information and helper functions"
+            "omit generating some runtime information and helper functions",
+            "Adjusts the compiler to implement D as a $(LINK2 $(ROOT_DIR)spec/betterc.html, better C):
+            $(UL
+                $(LI Predefines `D_BetterC` $(LINK2 $(ROOT_DIR)spec/version.html#predefined-versions, version).)
+                $(LI $(LINK2 $(ROOT_DIR)spec/expression.html#AssertExpression, Assert Expressions), when they fail,
+                call the C runtime library assert failure function
+                rather than a function in the D runtime.)
+                $(LI $(LINK2 $(ROOT_DIR)spec/arrays.html#bounds, Array overflows)
+                call the C runtime library assert failure function
+                rather than a function in the D runtime.)
+                $(LI $(LINK2 spec/statement.html#final-switch-statement/, Final switch errors)
+                call the C runtime library assert failure function
+                rather than a function in the D runtime.)
+                $(LI Does not automatically link with phobos runtime library.)
+                $(UNIX
+                $(LI Does not generate Dwarf `eh_frame` with full unwinding information, i.e. exception tables
+                are not inserted into `eh_frame`.)
+                )
+                $(LI Module constructors and destructors are not generated meaning that
+                $(LINK2 $(ROOT_DIR)spec/class.html#StaticConstructor, static) and
+                $(LINK2 $(ROOT_DIR)spec/class.html#SharedStaticConstructor, shared static constructors) and
+                $(LINK2 $(ROOT_DIR)spec/class.html#StaticDestructor, destructors)
+                will not get called.)
+                $(LI `ModuleInfo` is not generated.)
+                $(LI $(LINK2 $(ROOT_DIR)phobos/object.html#.TypeInfo, `TypeInfo`)
+                instances will not be generated for structs.)
```

Comment:
I'm not so sure that the CLI ages are the best place to document this. Imho we should just link to https://dlang.org/spec/betterc.html and improve this page if needed.

Reply:
I agree, but let's leave that to another PR.